sofiadinizms/NFT-onepage
# Feature/final tests

## What I did

1. Created the final project tests
2. Added app pics

## How to test

1. Run the project (yarn on the front, npm on the back)
2. Go to your local host 3001
4. Check if it's responsive
5. Check if the tests are passing and if they make sense

##Technical debts
1. NONE. ACABOU. É TETRAAA
